nightingale_navigation/configs/costmap_common_params.yaml

```diff
@@ -15,5 +16,5 @@ inflation_radius: 0.45
 obstacle_layer/footprint_clearing_enabled: true
 
 #raytrace_range must be less than max range of sensor.
-observation_sources: scan_multi
-scan_multi: {data_type: LaserScan, topic: movo/scan_multi, marking: true, clearing: true, inf_is_valid: true, obstacle_range: 4.75, raytrace_range: 8.0}
+observation_sources: base_scan_filtered
```

Comment:
What was the decision to change to base_scan_filtered?

Reply:
the laser scanner kept reporting a flickering obstacle in the hallway when the robot was in the room. the obstacle was not real bc the robot was in the lab and cannot see into the hallway. If i had to guess, id say the culprit is the reflective wheel on the desk the robohub folks just added to the room. the presence of this fake flickering point caused us to take the "long way around" to the stocking area. it was transient so didnt happen every time. 

anyway, the filtered scan rejects this lone, flickering point.

Reply:
the flickering point exists in the unfiltered scan, but does not exist in the filtered scan